client/scss/components/_figure.scss

```diff
@@ -54,7 +54,45 @@
   }
 }
 
-.figure--sticky {
-  position: sticky;
-  top: $vertical-space-unit;
+$large-width: (5/7 * 100)+'%';
+
+@include respond-to('medium') {
+  .figure--full-width {
+    margin-left: -18%;
```

Comment:
Is 18% the width of a gutter at this breakpoint? Did we have a formula for working that out somewhere, or did I imagine it?